modules/apps/jaggery/emm/src/emm/units/device-detail/device-detail.js

```diff
@@ -1,13 +1,32 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
 function onRequest(context) {
-    var log = new Log("device-detail.js");
+    // var log = new Log("device-detail.js");
```

Comment:
Thank you for pointing this. This file is moved to carbon-device-mgt-plugins repo and this is fixed there.